python/ray/tests/test_output.py

```diff
@@ -575,14 +575,20 @@ def test_disable_driver_logs_breakpoint():
 @ray.remote
 def f():
     while True:
-        time.sleep(1)
+        start_time = time.time()
```

Comment:
what is this change?

Reply:
This test failed on one of the commits. I was not able to reproduce locally on my Mac but I think it is because `time.sleep()` could be inaccurate on a VM. So the update here is to use `time.time()` to make the sleeping duration more accurate.